scripts/pg-drop

```diff
@@ -43,28 +48,26 @@ def get_dropstat_dir():
 
 class PgDropStat(object):
 
-    def __init__(self):
-        self.counters_db = SonicV2Connector(host='127.0.0.1')
-        self.counters_db.connect(self.counters_db.COUNTERS_DB)
-
-        self.configdb = ConfigDBConnector()
+    def __init__(self, namespace):
+        self.namespace = namespace
+        self.ns_list = multi_asic.get_namespace_list(namespace)
+        self.configdb = ConfigDBConnector(namespace=namespace, use_unix_socket_path=True)
```

Comment:
Let's not use unix_sockets everywhere as this will require users with elevated privilege to run this show command.